protocol/chat.go

```diff
@@ -89,6 +106,13 @@ type Chat struct {
 	// push notifications for this chat
 	Muted bool `json:"muted"`
 
+	// Time in which chat was muted
+	MuteTill int64 `json:"mutedTill,omitempty"`
```

Comment:
I would name the json and the field the same, now they are: `MuteTill` and `mutedTill`, so I would pick one or the other (`mutedUntil`, or `mutedTill` sounds better)